modules/37_industry/subsectors/declarations.gms

```diff
@@ -22,6 +22,7 @@ Parameters
   pm_IndstCO2Captured(ttot,all_regi,all_enty,all_enty,secInd37,all_emiMkt) "Captured CO2 in industry by energy carrier, subsector and emissions market"
 
   p37_CESMkup(ttot,all_regi,all_in)  "CES markup cost parameter [trUSD/CES input]"
+  p37_chemicals_feedstock_share(ttot,all_regi)   "minimum share of feso/feli/fega in total chemicals FE input [0-1]"
```

Comment:
I assume that this causes a compilation error as this parameter is also declared in datainput.gms. I would delete it here